modules/op-constructing-pipelines-using-pipeline-builder.adoc

```diff
@@ -27,32 +27,39 @@ The *Pipeline builder* view supports a limited number of fields whereas the *YAM
 +
-.. Click *Add Resources* to specify the name and type of resources that the pipeline run will use. These resources are then used by the tasks in the pipeline as inputs and outputs.
-For this example:
+.. In the *Resources* section, click *Add Resources* to specify the name and type of resources for the pipeline run. These resources are then used by the tasks in the pipeline as inputs and outputs. For this example:
 ... Add an input resource. In the *Name* field, enter `Source`, and then from the *Resource Type* drop-down list, select *Git*.
 ... Add an output resource. In the *Name* field, enter `Img`, and then from the *Resource Type* drop-down list, select *Image*.
-.. Optional: The *Parameters* for a task are prepopulated based on the specifications of the task. If required, use the *Add Parameters* link to add additional parameters.
 +
-.. A *Missing Resources* warning is displayed on a task if the resources for the task are not specified. Click the *s2i-nodejs* task to see the side panel with details for the task.
-+
-.Tasks details in Pipelines Builder
-image::op-pipeline-builder-task-details.png[]
+[NOTE]
+====
+A red icon appears next to the task if a resource is missing.
+====
 +
-.. In the task side panel, specify the resources and parameters for the *s2i-nodejs* task:
+.. Optional: The *Parameters* for a task are prepopulated based on the specifications of the task. If required, use the *Add Parameters* link in the *Parameters* section to add additional parameters.
+
+.. In the Workspaces section, click *Add workspace* and enter a unique workspace name in the Name field. You can add multiple workspaces to the pipeline.
+
+.. In the *Tasks* section, click the *s2i-nodejs* task to see the side panel with details for the task. In the task side panel, specify the resources and parameters for the *s2i-nodejs* task:
+
+... If required, in the *Parameters* section, add more parameters to the default ones, by using the $(params.<param-name>) syntax.
+... In the *Image* section, enter `Img` as specified in the *Resources* section.
+... Select a workspace from the *source* drop-down under *Workspaces* section.
+
+.. Add resources, parameters, and workspaces to the *openshift-client* task.
 
-... In the *Input Resources* -> *Source* section, the *Select Resources* drop-down list displays the resources that you added to the pipeline. For this example, select *Source*.
-... In the *Output Resources* -> *Image* section, click the *Select Resources* list, and select *Img*.
-... If required, in the *Parameters* section, add more parameters to the default ones, by using the `$(params.<param-name>)` syntax.
-... Similarly, add an input resource for the *openshift-client* task.
 . Click *Create* to create and view the pipeline in the *Pipeline Details* page.
-. Click the *Actions* drop-down menu, and then click *Start* to start the Pipeline.
+
+. Click the *Actions* drop-down menu, then click Start, to see the *Start Pipeline* page.
```

Comment:
'Start' must be in bold
s/ menu, then click Start, to see the *Start Pipeline* page./ menu, and then click **Start** to see the *Start Pipeline* page.